library/psa_crypto_pake.c

```diff
@@ -840,7 +865,10 @@ psa_status_t psa_pake_abort(psa_pake_operation_t * operation)
     {
         operation->input_step = PSA_PAKE_STEP_INVALID;
         operation->output_step = PSA_PAKE_STEP_INVALID;
-        operation->password = MBEDTLS_SVC_KEY_ID_INIT;
+        mbedtls_platform_zeroize( operation->password, operation->password_len );
```

Comment:
[We can't assume that `mbedtls_platform_zeroize(NULL, 0)` works](https://github.com/Mbed-TLS/mbedtls/issues/6633). So this needs a check on `password` or `password_len`. (Whereas `mbedtls_free(NULL)` is a well-defined no-op.)